wrappers/java/wrapper

```diff
@@ -33,23 +33,27 @@ wrapper_dir=$(dirname "$0")
 cd "$wrapper_dir"
 
 if [ "$action" == "version" ]; then
-	exec ant -S -q version
+	exec mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\['
```

Comment:
Having a pipe defeats the purpose of `exec`, which is making the wrapper return the exit code of the command (without having to do `exit $?`).
But the other issue with this solution is that it will log the downloads too, e.g.
```
Downloading: http://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-dependency-plugin/2.1/maven-dependency-plugin-2.1.jar
Downloaded: http://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-dependency-plugin/2.1/maven-dependency-plugin-2.1.jar (104 KB at 1992.3 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-release-plugin/2.0/maven-release-plugin-2.0.pom
Downloaded: http://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-release-plugin/2.0/maven-release-plugin-2.0.pom (8 KB at 331.9 KB/sec)
```
This other solution hides the download / build log 
```suggestion
	exec mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec
```